.changeset/chat-fixed-chat-window.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-chat/fixed-chat-window': patch
+---
+
+Fix storybook snapshot
```

Comment:
Do we need a changeset for this? I think I remember you bringing this up in a dev sync.